extend/index.md

```diff
@@ -10,15 +10,15 @@ You can extend and build on the Elastic Platform in several ways. Whether you ne
 
 There are two main approaches to extending Elastic: creating standardized Integrations for data ingestion or building custom solutions by developing directly against our REST APIs.
 
-## Path 1: Create an Elastic Integration
+## Path 1: Create an Elastic integration
 
-If your primary goal is to bring a new data source into the Elastic Stack in a standardized, reusable way, building an Elastic Integration is the best path. Integrations are pre-packaged assets that make it simple for users to collect and process data from a specific service or platform.
+If your primary goal is to bring a new data source into the Elastic Stack in a standardized, reusable way, building an Elastic integration is the best path. Integrations are pre-packaged assets that make it efficient for users to collect and process data from a specific service or platform.
 
 This approach is ideal for shipping data, logs, metrics, and traces. By building an integration, you contribute to the ever-growing ecosystem that both you and the community can benefit from.
 
-* Integrations are built on a consistent framework, providing users with a turnkey experience, including dashboards, visualizations, and alerts.  
+* We build integrations on a consistent framework, providing users with a turnkey experience, including dashboards, visualizations, and alerts.  
```

Comment:
"we" is not right here. it indicates that integrations provide consistency by leveraging a framework, but integrations aren't always built by "us" - that's why we have an extensibility guide for it